tests/functional/spec/census-household.spec.js

```diff
@@ -83,7 +83,7 @@ describe('Census Household', function () {
 
         // who-lives-here
         PermanentOrFamilyHome.clickPermanentOrFamilyHomeAnswerYes().submit()
-        HouseholdComposition.setPersonName(0, 'John Smith').submit()
+        HouseholdComposition.setFirstName(0, 'John').submit()
```

Comment:
Don't think we need to. We need to test the scenario that the middle/last is optional. Maybe that would be good as an explicit test. As for the second test I am going to delete, it's not really a selenium test, it is unit tested